src/schema/core.yaml

```diff
@@ -92,35 +92,57 @@ classes:
       - name
       - description
       - alternative_identifiers
-    slot_usage:
-      id:
-        structured_pattern:
-          todos:
-          syntax: "{id_nmdc_prefix}:{id_typecode_named_thing}-{id_shoulder}-{id_blade}{id_version}{id_locus}"
-          interpolated: true
-          partial_match: false
+  #    slot_usage:
+  #      id:
+  #        structured_pattern:
+  #          todos:
+  #          syntax: "{id_nmdc_prefix}:{id_typecode_named_thing}-{id_shoulder}-{id_blade}{id_version}{id_locus}"
+  #          interpolated: true
+  #          partial_match: false
 
 
   MaterialEntity:
+    abstract: true
```

Comment:
Does abstract: true mean that it doesn't need a typecode? That was true for the other examples but is not how MaterialEntity is written.

Reply:
`abstract: true` means that the class shouldn't be instantiated directly. (Only subclasses of `MaterialEntity` should be instantiated. See https://microbiomedata.github.io/nmdc-schema/MaterialEntity/)

Since you can't directly instantiate `MaterialEntity`, you can't assign an `id`, so there's no sense in assigning a typecode.

I see that I **did** assign a typecode for   `MaterialEntity.slot_usage.id.pattern`. That makes things ore confusing, so I'll remove it now.

Reply:
I left `abstract: true` on `MaterialEntity` and removed the `id` pattern/typcode constraint, but this comment hasn't updated to "outdated"